packages/dht/src/connection/ConnectionManager.ts

```diff
@@ -584,15 +584,15 @@ export class ConnectionManager extends EventEmitter<Events> implements ITranspor
         return response
     }
 
-    // IConnectionLocker server implementation
+    // ConnectionLockRpc local RPC method
```

Comment:
Unify comments for server methods. here we have:
`// ConnectionLockRpc local RPC method`
but at some places we have e.g.
`IConnectionLocker implementation`